IPython/core/magics/script.py

```diff
@@ -229,20 +262,9 @@ def shebang(self, line, cell):
                 print("Error while terminating subprocess (pid=%i): %s" \
                     % (p.pid, e))
             return
-        out = py3compat.decode(out)
-        err = py3compat.decode(err)
-        if args.out:
-            self.shell.user_ns[args.out] = out
-        else:
-            sys.stdout.write(out)
-            sys.stdout.flush()
-        if args.err:
-            self.shell.user_ns[args.err] = err
-        else:
-            sys.stderr.write(err)
-            sys.stderr.flush()
         if args.raise_error and p.returncode!=0:
-            raise CalledProcessError(p.returncode, cell, output=out, stderr=err)
+            print(p.returncode)
```

Comment:
missing debug ?

Reply:
Whoops, that might be something I flipped when I was debugging and forgot to out back. Lemme take a look.